app/api/resources/users.py

```diff
@@ -47,13 +48,18 @@ def post(cls):
 
         data = request.json
 
+        is_valid = validate_user_registration_request_data(data)
+
+        if is_valid != {}:
+            return is_valid, HTTPStatus.BAD_REQUEST
+
         # send POST /register request to MS API and return response
```

Comment:
Line gaps seem inconsistent.

Reply:
yep, fixed it now.

Reply:
Is this ok for code readability? since the line shows separation of logic. or should I put all logic together without line gaps?

<img width="708" alt="Screen Shot 2020-06-22 at 1 02 41 am" src="https://user-images.githubusercontent.com/29667122/85228023-40f76980-b424-11ea-9198-2f54bc729971.png">